src/snapshot/helpers.js

```diff
@@ -15,6 +15,7 @@ exports.getDelay = function(fullLayout) {
     return (
         fullLayout._has('gl3d') ||
         fullLayout._has('gl2d') ||
+        fullLayout._has('sankey') ||
```

Comment:
Why do we need this?

Reply:
The opacity of links is now zero on `enter()` which is why we need to wait a bit here. I understand this is not a super clean solution and I could either:
- remove animations altogether
- remove animations when saving to image

Reply:
Oh we have an animation on `enter()` now. Was this a requirement?

If not, I'd vote :hocho: that out for now. Animation on first-draw is something that we'll need to add at some point, but we should do that for traces, not just for `sankey`.

Reply:
Out of curiosity, is there an easy way to tell that we're in the process of creating an image and force the opacity of nodes and links to be `1` if that's the case?

Reply:
You can use `gd._context.staticPlot`

Reply:
I removed the delay in `src/snapshot/helpers.js` while still passing the image tests

See commit b90fda1 and b4d7ff3